frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -4,57 +4,17 @@ import { browser, $, $$, by, ExpectedConditions as until, Key, element } from 'p
 import { safeLoad, safeDump } from 'js-yaml';
 import * as _ from 'lodash';
 import { execSync } from 'child_process';
-import { OrderedMap } from 'immutable';
 
 import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
 import * as crudView from '../views/crud.view';
 import * as yamlView from '../views/yaml.view';
-import * as namespaceView from '../views/namespace.view';
 import * as createRoleBindingView from '../views/create-role-binding.view';
 
 const K8S_CREATION_TIMEOUT = 15000;
 
 describe('Kubernetes resource CRUD operations', () => {
   const testLabel = 'automatedTestName';
   const leakedResources = new Set<string>();
-  const k8sObjs = OrderedMap<string, { kind: string; namespaced?: boolean }>()
```

Comment:
This will remove these tests from our release-informing scenario here:

https://github.com/openshift/console/blob/master/frontend/integration-tests/protractor.conf.ts#L104

https://github.com/spadgett/release/blob/7db78089a1784208fd1fdffc7ffd729444533888/ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml#L4769

We need a way to selectively run a subset of Cypress tests and add this to the periodic jobs before we move these tests out.

Reply:
ok, added this to the PR's todo list.